lamindb_setup/_check_setup.py

```diff
@@ -91,17 +126,19 @@ def _check_instance_setup(from_module: str | None = None) -> bool:
         if (
             from_module is not None
             and settings.auto_connect
-            and not django.IS_SETUP
+            and not django_lamin.IS_SETUP
             and not IS_LOADING
         ):
-            if not from_module == "lamindb":
+            if from_module != "lamindb":
+                _check_in_modules(from_module, isettings)
```

Comment:
How long does this run?

Can we add a comment and/or a more descriptive function name?

Reply:
done